CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 #### Fixes
 
+* [#2222](https://github.com/ruby-grape/grape/pull/2222): Fix autoload types and validators - [@ericproulx](https://github.com/ericproulx).
```

Comment:
"Fix" is not needed here.

This is more like "a second attempt of autoload types and validators" :) but it would be enough just remove "Fix"